docs/02-app/01-building-your-application/01-routing/03-linking-and-navigating.mdx

```diff
@@ -184,6 +184,58 @@ For a full list of `useRouter` methods, see the [API reference](/docs/app/api-re
 
 > **Recommendation:** Use the `<Link>` component to navigate between routes unless you have a specific requirement for using `useRouter`.
 
+## `redirect` function
```

Comment:
```suggestion
## `redirect` Function
```

Reply:
@ztanner It was lowercase elsewhere, so I'll fix that too then

Reply:
Actually, it's lowercase in most places <picture data-single-emoji=":thinkies:" title=":thinkies:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/thinkies/d164ba479fed9c0d.png" alt=":thinkies:" align="absmiddle"></picture>